pos_deposit/models/product_template.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Sunflower IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
```

Comment:
Hi. 
I don't think that it is a ``product.template`` but a ``product.product`` value. Typical Use cas : 
- product template : Cola
- variant 1 : 25cl ; variant 2 : 50 cl. 
-> variant 1 : deposit product = bottle 25cl ; variant 2 : deposit product = 50cl.

don't you think ? 

then we can implement a related field on template (as per other fields like ``barcode`` to make it available on template form, in a single variant context). 

Don't you think ?